src/webextension/widgets/input.css

```diff
@@ -22,7 +22,7 @@
 
 .input[disabled],
 .input-wrapper.disabled {
-  background-color: #ededf0;
+  opacity: 0.7;
```

Comment:
I'm a little worried about this, since it might have contrast issues (especially since the color underneath will "bleed through" the input now). @kimberlythegeek Could you take a look to make sure we're not doing bad things with our contrast here?